game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/TotalPowerAndTotalRolls.java

```diff
@@ -466,6 +475,38 @@ static void sortAaHighToLow(
             data.getUnitTypeList().getSupportRules(),
             !defending,
             false);
+
+    return getUnitPowerAndRollsForNormalBattles(
+        unitsGettingPowerFor,
+        enemySupport,
+        friendlySupport,
+        defending,
+        data,
+        Matches.territoryIsLand().test(location),
+        territoryEffects,
+        unitSupportPowerMap,
+        unitSupportRollsMap);
+  }
+
+  @VisibleForTesting
+  static Map<Unit, TotalPowerAndTotalRolls> getUnitPowerAndRollsForNormalBattles(
```

Comment:
Codacy found an issue: [The method 'getUnitPowerAndRollsForNormalBattles(Collection, SupportCalculationResult, SupportCalculationResult, boolean, GameData, boolean, Collection, Map, Map)' has an NPath complexity of 224, current threshold is 200](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6303580)